loopout_test.go

```diff
@@ -90,6 +91,12 @@ func TestLoopOutPaymentParameters(t *testing.T) {
 		t.Fatal("unexpected state")
 	}
 
+	// check that the SwapInfo contains the outgoing chan set
```

Comment:
style nit: comments should be sentences (starting capitalized, ending in punctuation).